File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/bo/RolePermBO.java
package ink.ckx.mo.system.api.model.bo;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/16
 */
@Schema(description = "角色权限BO")
@Data
public class RolePermBO {

    @Schema(description = "权限标识")
    private String perm;

    @Schema(description = "角色编码")
    private String code;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/bo/RolePermBO.kt
package ink.ckx.mo.admin.api.model.bo

import io.swagger.v3.oas.annotations.media.Schema

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/16
 */
@Schema(description = "角色权限BO")
data class RolePermBO(

    @Schema(description = "权限标识")
    var perm: String? = null,

    @Schema(description = "角色编码")
    var code: String? = null
)